src/components/Blogs.tsx

```diff
@@ -0,0 +1,141 @@
+      property: "طراحی اختصاصی",
+      date: "16 فروردین 1403 ",
+      imageUrl: "/4e9dc13f8cc7a1e596d4ae3efcf5090d24dd6975.png",
+
+      link: "/",
+    },
+    {
+      title: "۳. چرا انتخاب محصولات سفارشی ارزشمند است؟",
+      property: "طراحی اختصاصی",
+      date: "16 فروردین 1403 ",
+      imageUrl: "/a9673fa90ca5cef9574e5ae8d9e06c05c53e93d4.png",
+      link: "/",
+    },
+    {
+      title: "۳. چرا انتخاب محصولات سفارشی ارزشمند است؟",
+      property: "طراحی اختصاصی",
+      date: "16 فروردین 1403 ",
+      imageUrl: "/ed717e5db8432410cbf35a4a7d65c72422c44212.png",
+      link: "/",
+    },    {
+      title: "۳. چرا انتخاب محصولات سفارشی ارزشمند است؟",
+      property: "طراحی اختصاصی",
+      date: "16 فروردین 1403 ",
+      imageUrl: "/ed717e5db8432410cbf35a4a7d65c72422c44212.png",
+      link: "/",
+    },    {
+      title: "۳. چرا انتخاب محصولات سفارشی ارزشمند است؟",
+      property: "طراحی اختصاصی",
+      date: "16 فروردین 1403 ",
+      imageUrl: "/ed717e5db8432410cbf35a4a7d65c72422c44212.png",
+      link: "/",
+    },    {
+      title: "۳. چرا انتخاب محصولات سفارشی ارزشمند است؟",
+      property: "طراحی اختصاصی",
+      date: "16 فروردین 1403 ",
+      imageUrl: "/ed717e5db8432410cbf35a4a7d65c72422c44212.png",
+      link: "/",
+    },
+  ];
+  const BlogCard = ({ blog }: { blog: BlogType }) => {
+    return (
+      <Link
+        href={blog.link}
+        className="embla__slide grow-0 shrink-0 basis-[70%] flex flex-col gap-8 mx-2 md:basis-[calc(100%/2)] lg:basis-[calc(100%/4-16px)]"
+        draggable="false"
+      >
+        <Image
+          alt={blog.title}
+          src={blog.imageUrl}
+          className="object-cover rounded-t-[150px] rounded-b-lg h-80 w-full"
+          width={321}
+          height={321}
+          draggable="false"
+        />
+        <div className="flex justify-between items-center flex-row w-full text-gray-400 text-xs xl:text-sm 2xl:text-lg">
+          <div className="flex items-center lg:gap-2 ">
+            <FolderIcon className="w-3 h-3 sm:w-4 sm:h-4 md:w-6 md:h-6 " />
+            <span>{blog.property}</span>
+          </div>
+          <div className="flex items-center gap-2">
+            <ClockIcon className="w-3 h-3 sm:w-4 sm:h-4 md:w-6 md:h-6 "/>
+            <span>{blog.date}</span>
+          </div>
+        </div>
+        <div className="w-full text-sm md:text-lg lg:text-2xl font-bold">
+          {blog.title}
+        </div>
+      </Link>
+    );
+  };
+
+  const Blogs = () => {
+    const [emblaRef, emblaApi] = useEmblaCarousel({
+      containScroll: "trimSnaps",
+      direction: "rtl",
+      slidesToScroll: "auto",
+      loop: true,
+      align:'center'
+    });
+
+    return (
+      <section className="rounded-3xl w-full flex flex-col relative gap-12 mt-60 mb-64">
+        <div className="w-full flex justify-between h-18 items-center ">
+          <b className="text-2xl max-md:m-auto md:text-4xl">مقالات اخیر</b>
```

Comment:
using **Heading** Tag for example: `h1`, `h2`, `h3`
I have suggestion for you: using `h2`